### ISSUE TITLE
Deleting last event does not refresh page

### ISSUE THREAD

**Issue by Germs2004** [ZoneMinder/ZoneMinder#488]
1) Go to the Event list for a certain monitor that has at least 1 event listed.
2) Select the last event to view it
3) Click the Delete button to delete it
4) Notice that the page still shows the same Event ID and other details even after it's deleted.

When you delete the last event, I think the system should either move backwards to the previous event, loop forwards to the first event, or the best solution: show a message saying that there are no more events and remove all mention of the event you just deleted.

The same should happen if I delete the one and only event; it should show a page saying that there are no more events.

Basically, make it clear to the user that the Delete button actually worked and the video they were looking at is deleted.

**Comment by knnniggett** [comment]
@Germs2004 I can agree with your statement that this part of zoneminder could be made better. However, we don't currently have the resources at this time to update non-critical issues such as this one.  Naturally, if any one else desires to fix this on their own, a pull request would be welcome.

As a long term goal, we plan to release a new, cakephp-based web portal, which will not have usability issues such as this one.
